dashboard-ui/src/pages/console/viewer.tsx

```diff
@@ -150,7 +155,9 @@ export const useViewerFacets = () => {
 
   // Update nodes facet
   nodes.forEach((node) => {
-    nodeCounts.set(node.metadata.name, 0);
+    if (node?.metadata?.name) {
```

Comment:
This type guard shouldn't be necessary (`node.metadata.name` isn't undefined/nullable)